devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -61,18 +63,57 @@ metadata {
 		main "motion"
 		details(["motion", "battery", "tamper"])
 	}
-}
+
+	// Preferences for Everspring SP815 and Everspring SP817
+	// preferences {
```

Comment:
I think we should just stick with the fingerprints and not expose the preferences.

Reply:
@greens 
Changes in this PR cover integrations: WWST-5069 and WWST-5131.
We agreed (in ICP-12456) with @tpmanley and Eric about exposing few preferences for those devices in this DTH.
Since those both Everspring devices fall asleep quite quickly - we need to deal with configuring them in a tricky way.

Reply:
retrigger interval is fine, but the reporting interval may lead to the user shooting themselves in the foot by killing their device's battery life. Could we raise the minimum to at least 10 minutes?

Reply:
ok, it is fixed now